features/FEATURE_BLE/ble/generic/GenericGap.h

```diff
@@ -302,10 +335,12 @@ class GenericGap : public ::Gap,
 
     void on_unexpected_error(const pal::GapUnexpectedErrorEvent &e);
 
-    pal::own_address_type_t get_own_address_type();
+    pal::own_address_type_t get_own_address_type(bool central_not_peripheral, bool non_connectable_or_scan_request);
```

Comment:
github is deleting my comments??!

I was suggesting to replace these two bool with an enum, especially the naming of the first one is awkward so that would fix it:
  enum {
    CENTRAL_CONNECTABLE,
    CENTRAL_NON_CONNECTABLE_OR_SCAN,
    PERIPHERAL_CONNECTABLE,
    PERIPHERAL_NON_CONNECTABLE_OR_SCAN
  }

Reply:
Good point, that's much better :)